composer.json

```diff
@@ -1,40 +1,47 @@
 {
-    "name": "phpro/zf-doctrine-hydration-module",
-    "license": "MIT",
-    "description": "Doctrine hydrators for ZF2",
-    "keywords": [
-        "zf2",
-        "doctrine",
-        "hydrator"
-    ],
-    "authors": [
-        {
-            "name": "Toon Verwerft",
-            "email": "toon.verwerft@phpro.be"
-        }
-    ],
-    "require": {
-        "php": ">=5.3.23",
-        "doctrine/common": "~2.1",
-        "zendframework/zend-modulemanager": "~2.0",
-        "zendframework/zend-servicemanager": "~2.0",
-        "zendframework/zend-hydrator": "~1.0 || ^2.0",
-        "api-skeletons/zf-doctrine-module-zend-hydrator": "^1.0",
-        "doctrine/doctrine-module": "~1.0",
-        "doctrine/instantiator": "~1.0.4"
-    },
-    "require-dev": {
-        "fabpot/PHP-CS-Fixer": "~1.10",
-        "phpunit/phpunit": "~4.8",
-        "doctrine/doctrine-orm-module": "~0.9",
-        "doctrine/doctrine-mongo-odm-module": "~0.9",
-        "doctrine/mongodb-odm": "~1.1",
-        "phpro/grumphp": "~0.7"
-    },
-    "autoload": {
-      "psr-4": {
-        "Phpro\\DoctrineHydrationModule\\": "src/",
-        "PhproTest\\DoctrineHydrationModule\\": "test/src/"
-      }
+  "name": "phpro/zf-doctrine-hydration-module",
+  "license": "MIT",
+  "description": "Doctrine hydrators for ZF2 and ZF3",
+  "keywords": [
+    "zf2",
+    "doctrine",
+    "hydrator"
+  ],
+  "authors": [
+    {
+      "name": "Toon Verwerft",
+      "email": "toon.verwerft@phpro.be"
     }
+  ],
+  "extra": {
+    "zf": {
+      "module": "Phpro\\DoctrineHydrationModule"
+    }
+  },
+  "require": {
+    "php": "^5.6 || ^7.0",
+    "doctrine/common": "^2.6.1",
+    "zendframework/zend-servicemanager": "^2.7.6 || ^3.1",
+    "zendframework/zend-hydrator": "^1.1 || ^2.2.1",
+    "api-skeletons/zf-doctrine-module-zend-hydrator": "^1.0",
+    "doctrine/doctrine-module": "^1.2",
+    "doctrine/instantiator": "^1.0.5"
+  },
+  "require-dev": {
+    "fabpot/PHP-CS-Fixer": "^1.11.6",
+    "phpunit/phpunit": "^4.8",
+    "zendframework/zend-modulemanager": "^2.7.2",
```

Comment:
Moved from `require-dev` to `require` section in composer 939f9c8cfc7d1f08b7b918947764877a737b8666